modules/jupyter-widget/src/utils.js

```diff
@@ -60,3 +78,30 @@ export function initDeck({mapboxApiKey, container, jsonInput}, onComplete, handl
     });
   });
 }
+
+function tabularize(json) {
+  const dataTable = document.createElement('div');
+  dataTable.className = 'dataTable';
+
+  for (const key in json) {
+    const row = document.createElement('div');
+    const header = document.createElement('div');
+    header.className = 'header';
+    header.innerHTML = key;
+    header.style.fontWeight = 500;
+    header.style.marginRight = '10px';
+    const value = document.createElement('div');
+    value.className = 'value';
+    value.innerHTML = JSON.stringify(json[key]);
```

Comment:
Use `innerText` instead of `innerHTML` - user data may contain contaminated content.